application/common/config/local.test.php

```diff
@@ -1,6 +1,43 @@
 <?php
 
 return [
+    'params' => [
+        'password' => [
+            'minLength' => [
+                'value' => 10,
+                'phpRegex' => '',
+                'jsRegex' => '.{10,}',
+                'enabled' => true
+            ],
+            'maxLength' => [
+                'value' => 255,
+                'phpRegex' => '',
+                'jsRegex' => '.{0,255}',
+                'enabled' => true
+            ],
+            'minNum' => [
+                'value' => 2,
+                'phpRegex' => '',
+                'jsRegex' => '(\d.*){2,}',
```

Comment:
Have we tested this regex? I don't think it's actually enforcing what we want it to.